doc/migrations/3.1.1_3.1.1++.md

```diff
@@ -10,3 +10,26 @@
 ## **`ejabberd.cfg` renamed to `mongooseim.cfg`**
 
 Rename the existing config file of MongooseIM from `ejabberd.cfg` to `mongooseim.cfg`.
+
+## Pools configuration
+
+The way how pools to external service are configured changed, please see [Outgoing Connection doc](../advanced-configuration/outgoing-connections.md) for more details.
+
+NOTE: Keep in mind that `outgoing_pools` is a list of pools, it may turn out that you will have more than one entry in the list when more than outgoing pool is needed.
```

Comment:
NOTE: Keep in mind that `outgoing_pools` is a list of pools, it may turn out that you will have more than one entry in the list when more than a single outgoing pool is needed.